package.json

```diff
@@ -75,10 +75,10 @@
 		"express-ipfilter": "^1.1.2",
 		"feedme": "^1.2.0",
 		"helmet": "^3.23.3",
-		"ical": "^0.8.0",
-		"iconv-lite": "^0.6.2",
+		"iconv-lite": "latest",
 		"module-alias": "^2.2.2",
-		"moment": "^2.28.0",
+		"moment": "latest",
```

Comment:
See above

Reply:
probably accidentally. 

remove ical, add node-ical  was all I was trying to do

I don't think any hacks can be removed.. I think its pretty clean now